mobile/I10n.yaml

```diff
@@ -0,0 +1,3 @@
+arb-dir: lib/l10n
+template-arb-file: app_en.arb
+output-localization-file: app_localizations.dartà
```

Comment:
"à" at the end of the file

Reply:
Will be fixed in the Translation-module PR